src/python/T0/RunConfig/Tier0Config.py

```diff
@@ -789,6 +789,15 @@ def addRepackConfig(config, streamName, **options):
     else:
         streamConfig.Repack.BlockCloseDelay = options.get("blockCloseDelay", 24 * 3600)
 
+    if hasattr(streamConfig.Repack, "SetStreamMemory"):
+        streamConfig.Repack.SetStreamMemory = options.get("setStreamMemory", streamConfig.Repack.SetStreamMemory)
+    else:
+        streamConfig.Repack.SetStreamMemory = options.get("setStreamMemory", 2000)
+
+    if hasattr(streamConfig.Repack, "SetConfStreamMemory"):
+        streamConfig.Repack.SetConfStreamMemory = options.get("setConfStreamMemory", streamConfig.Repack.SetConfStreamMemory)
```

Comment:
As Dima already commented, this attribute is not necessary. We don't need a list of streams to modify, as what we will do is add a new configuration for each stream that needs a configuration different from the default.